packages/archway-test-tube/libarchway/testenv/setup.go

```diff
@@ -202,6 +202,7 @@ func InitChain(appInstance *app.ArchwayApp) (sdk.Context, secp256k1.PrivKey) {
 	requireNoErr(err)
 
 	concensusParams := simtestutil.DefaultConsensusParams
+	// TODO: max gas could be set to 300mil
```

Comment:
So, why not..? 😅

Reply:
I wanted to leave it for another issue since its not the context of this PR